source/help/getting-started/organizing-conversations.rst

```diff
@@ -46,40 +46,54 @@ Channels can be created, joined, renamed, left, marked as a favorite, muted, unm
 Creating a channel
 ~~~~~~~~~~~~~~~~~~~
 
-Create a new Public channel or Private channel by selecting the **+** symbol next to the **Public Channels*** or **Private Channels** header on the left hand navigation. To start a Direct Message thread, select the **+** symbol next to the **Direct Messages** bar or select **More** at the bottom of the list to view team members you can message.
-
-    .. image:: ../../images/Create_private_channel.png
+Create a new Public or Private channel by selecting the **+** symbol at the top of the sidebar, next to the **History** arrows, then selecting **Create New Channel**.
 
+    .. image:: ../../images/create-new-channel.png
+    
 Anyone can create Public channels or Private channels, unless the System Admin has `restricted the permissions <https://docs.mattermost.com/administration/config-settings.html#enable-public-channel-creation-for>`__.
 
+There are two ways to start a Direct or Group Message:
+
+Select the **+** symbol at the top of the sidebar, next to the **History** arrows, then select **Open a Direct Message**.
```

Comment:
Select the **+** symbol at the top of the sidebar, then select **Open a Direct Message**